Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -19,25 +20,28 @@ class TransactionHistoryViewController: UIViewController {
         return refreshControl
     }()
     
-    @IBOutlet weak var syncLabel: UILabel!
     
+    @IBOutlet weak var syncLabel: UILabel!
     @IBOutlet var tableView: UITableView!
     @IBOutlet var btnFilter: DropMenuButton!
-    var FromMenu = true
-    var visible:Bool = false
     
-    var filterMenu = [LocalizedStrings.all] as [String]
-    var filtertitle = [0] as [Int]
+    var filterMenu = [LocalizedStrings.all]
```

Comment:
`filterMenu`? I think `filterOptions` or `txFilterOptions` are better names. `menu` does not suggest a list (array) of items.

Reply:
fixed in 9a8c24b